lib/Doctrine/ORM/AbstractQuery.php

```diff
@@ -583,21 +583,45 @@ public function getResultCacheDriver()
      * Set whether or not to cache the results of this query and if so, for
      * how long and which ID to use for the cache entry.
      *
-     * @param boolean $bool
+     * @deprecated 3.0 Use {@see enableResultCache} and {@see disableResultCache} instead.
```

Comment:
```suggestion
     * @deprecated Will be removed in 3.0. Use {@see enableResultCache} and {@see disableResultCache} instead.
```

In case this will be removed in 3.0.

Reply:
@SenseException https://docs.phpdoc.org/references/phpdoc/tags/deprecated.html

Reply:
TIL. Is there tooling that takes this into account? Symfony's profiler maybe?

Reply:
I have no idea. Anyway, IMO it's at least future-proof to stick to existing standard.

Reply:
I've looked into other Doctrine projects and found a few deprecation entries that follow the documented phpdoc pattern. I think we can keep it as you proposed in your PR.